fairlearn/experimental/enable_metric_frame_plotting.py

```diff
@@ -0,0 +1,17 @@
+"""Enables Metric Frame Plotting
```

Comment:
You're probably getting some kind of `flake8` warning for this for missing the copyright.